code/game/machinery/Sleeper.dm

```diff
@@ -5,10 +5,10 @@
 	density = FALSE
 
 /obj/machinery/sleeper
-	name = "sleeper"
-	desc = "An enclosed machine used to stabilize and heal patients."
-	icon = 'icons/obj/machines/sleeper.dmi'
-	icon_state = "sleeper"
+	name = "autodoc"
+	desc = "An old pre war machine, used to stablize and heal patients"
+	icon = 'icons/obj/machines/autodoc.dmi'
+	icon_state = "autodoc"
```

Comment:
we likely want this to be above windows and other objects since it's a tall fella
using the layer variable we can set it to be above windows (who are at layer 3.2) by adjusting our layer to 3.3

otherwise we get clipping :)

```suggestion
	icon_state = "autodoc"
	layer = 3.3
```